lib/core/include/irods/rodsPackInstruct.h

```diff
@@ -59,7 +59,7 @@
 
 #define PortList_PI "int portNum; int cookie; int sock; int windowSize; str hostAddr[LONG_NAME_LEN];"
 
-#define PortalOprOut_PI "int status; int l1descInx; int numThreads; str chksum[NAME_LEN]; struct PortList_PI;"
+#define PortalOprOut_PI "int status; int l1descInx; int numThreads; str chksum[CHKSUM_LEN]; struct PortList_PI;"
```

Comment:
Is there any reason to think that changing a packing instruction in this way will break anything?

We might need to hold off of changing packing instructions until we know more.

Reply:
The proposed change is correct and good in theory, but we have no way of confirming the effects due to our limited knowledge of the PackStruct implementation.

Reply:
Based on everything I know about packstruct, this code change will not result in any behavioral changes and therefore will not impact compatibility. However, I accept that my understanding of packstruct is quite limited. I've only dug into it enough to be reasonably confident that this change won't break anything. But since we can't verify that, I'm fine with not including this change in this PR. (It looks like I missed one, anyway.) I'll make a note of this in #8721.